api/NetworkInformation.json

```diff
@@ -107,7 +107,8 @@
               "version_added": "38"
             },
             "edge": {
-              "version_added": false
+              "version_added": "79",
+              "notes": "Only supported in Edge OS"
```

Comment:
"Edge OS" hehe :)
Please remove this note.

Reply:
_Imagine if Microsoft DID make a web-only OS...  😛_ 

Should we also set the support itself to `false`?  If it's only supported in Chrome OS, seems like it wouldn't be supported in Edge at all.

Reply:
:D 
Good point, yes, I agree to set it to `false`.